_maps/RandomRuins/SpaceRuins/gasthelizards.dmm

```diff
@@ -136,20 +136,14 @@
 	dir = 1;
 	layer = 2.9
 	},
-/turf/open/floor/plasteel{
-	dir = 1;
-	icon_state = "blue"
-	},
+/turf/open/floor/plasteel,
 /area/ruin/space/has_grav/gasthelizard)
 "q" = (
 /obj/machinery/door/window/brigdoor/northleft,
 /obj/machinery/atmospherics/components/binary/pump{
 	dir = 1
 	},
-/turf/open/floor/plasteel{
-	dir = 1;
-	icon_state = "blue"
```

Comment:
![image](https://user-images.githubusercontent.com/6209658/45307663-98e20c00-b4ed-11e8-9f4b-54a3a7016c7c.png)
decals

Reply:
Ah, that explains it. Give me a minute